packages/fuels/package.json

```diff
@@ -55,7 +55,9 @@
     "build:package": "tsup",
     "build:browser": "pnpm vite build",
     "build:minified": "pnpm uglifyjs --compress --mangle --output dist/browser.min.mjs -- dist/browser.mjs",
-    "postbuild": "tsc --emitDeclarationOnly -p tsconfig.dts.json",
+    "postbuild": "run-s type:declarations type:check",
+    "type:declarations": "tsc --emitDeclarationOnly -p tsconfig.dts.json",
+    "type:check": "tsc --noEmit",
```

Comment:
Does this `tsc` follow the rules defined in `tsconfig.base.json`?

Reply:
Why wouldn't it?

It would have run checks on the `dist` and `node_modules` if it wasn't following the [rules](https://github.com/FuelLabs/fuels-ts/blob/c1bf8d75df35806f49d6226f774ce97ff227c7ca/tsconfig.base.json#L17) of `tsconfig.base.json` would it not?